### PR TITLE
Add some documentation/warning about JAAS's javax.security.auth.useSu…

### PR DESCRIPTION
**Pull request by joshelser**
…bjectCredsOnly property

### CODE REVIEW

**Comment by joewitt** [comment]
Huge fan of this very important life lesson and PR

File: sections/jaas.md [hunk 1]
```diff
@@ -60,3 +60,21 @@ Client {
...
+
+```
+-Djavax.security.auth.useSubjectCredsOnly=true
+```
+
+When this system property is set to `true` (the default), the `LoginModule` will never try to obtain credentials when they are not present in the context of the current call (e.g. the current `doAs` scope). When this property is set to `false`, the `LoginModule`
+has the ability to try to obtain credentials by `any other means`. This means that the implementation of the `LoginModule` has the "latitude" to do whatever it likes.
+
+For the `Krb5LoginModule` on Oracle Java, the implementation will attempt to obtain credentials via a prompt on standard input/output.
+When the provided JAAS configuration otherwise instructs the `Krb5LoginModule` to never prompt or use a ticket cache, this can a
```
Comment (steveloughran) [inline sections/jaas.md]: s/a/r/be a/

**Comment by steveloughran** [comment]
apart from the suggeetted edit, LGTM.

I think KDiag should consider doing a jump of all javax.security settings, if it doesn't already

**Comment by joshelser** [comment]
Thanks, @steveloughran. Pushed the missing word into 82201b6.

**Pull request merged**